filebeat/module/auditd/log/ingest/pipeline.json

```diff
@@ -86,10 +88,56 @@
                 }
             }
         },
+
+        { "rename": { "ignore_failure": true, "field": "auditd.log.arch", "target_field": "host.os.architecture" } },
+
+        { "rename": { "ignore_failure": true, "field": "auditd.log.acct", "target_field": "user.name" } },
+
+        { "rename": { "ignore_failure": true, "field": "auditd.log.gid", "target_field": "group.id" } },
```

Comment:
I like these 1-line rename statements. It reads a bit like a table now.

Reply:
Yes, and the subtle changes in column width hint at different processors or different things happening. :-)